From 93a85d2b6028d221bed573ce6bafacf4bb479005 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 6 Nov 2018 15:02:23 +0100
Subject: [PATCH] [SQUSH][DNM] Merge tag 'android-9.0.0_r16' into lineage-16.0

commit a0602572770c76e637ca1ac9a9cf62b0e7e6eb43
Merge: 8d29db3 bd8b17e
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Fri Aug 17 03:04:41 2018 +0000

    Snap for 4959688 from bd8b17e1032cc5fd029341d6f6a1917acf26f4d2 to pi-qpr1-release

    Change-Id: Ib92d8bd172f462ac2194c574ab563695657ce8ca

commit bd8b17e1032cc5fd029341d6f6a1917acf26f4d2
Author: Hall Liu <hallliu@google.com>
Date:   Thu Jul 26 11:09:12 2018 -0700

    Add auto-redial carrier config for DTAG

    Test: same as the others
    Bug: 111235453
    Change-Id: Id8d2216f1880d215903c33beaae2b77348ba56c5

commit 8d29db3ee481ced0a4df9831b1bff2642766e85c
Author: Amit Mahajan <amitmahajan@google.com>
Date:   Tue Jun 5 16:31:16 2018 -0700

    Add non-roaming registered PLMN name for Xfinity.

    Test: manual
    Bug: 80145031
    Change-Id: I637b36fd0fe58e1c11100496fedbb52acc67405c

Change-Id: I139186a8c6a1f9e8bb81e308bbcd2d324678a06b
---
 assets/carrier_config_26201.xml  | 1 +
 assets/carrier_config_311480.xml | 5 +++++
 2 files changed, 6 insertions(+)

diff --git a/assets/carrier_config_26201.xml b/assets/carrier_config_26201.xml
index 0c23873..8d94903 100644
--- a/assets/carrier_config_26201.xml
+++ b/assets/carrier_config_26201.xml
@@ -4,6 +4,7 @@
         <string name="default_vm_number_string">3311</string>
         <boolean name="display_hd_audio_property_bool" value="false" />
         <boolean name="carrier_name_override_bool" value="true" />
+        <boolean name="auto_retry_failed_wifi_emergency_call" value="true"/>
         <string name="carrier_name_string">Telekom</string>
     </carrier_config>
 </carrier_config_list>
diff --git a/assets/carrier_config_311480.xml b/assets/carrier_config_311480.xml
index d13ff54..7639005 100644
--- a/assets/carrier_config_311480.xml
+++ b/assets/carrier_config_311480.xml
@@ -156,4 +156,9 @@
         <boolean name="carrier_name_override_bool" value="true" />
         <string name="carrier_name_string"></string>
     </carrier_config>
+    <!-- XFINITY Mobile as operator name for Xfinity -->
+    <carrier_config gid1="BA01450000000000">
+        <boolean name="cdma_home_registered_plmn_name_override_bool" value="true" />
+        <string name="cdma_home_registered_plmn_name_string">XFINITY Mobile</string>
+    </carrier_config>
 </carrier_config_list>
-- 
2.17.1

